docs/.vuepress/theme/components/SearchBox.vue

```diff
@@ -1,35 +1,54 @@
 <template>
-  <div class="search-box">
-    <icon name="search" />
-    <input
-      @input="query = $event.target.value"
-      aria-label="Search"
-      :value="query"
-      :class="{ focused: focused }"
-      autocomplete="off"
-      spellcheck="false"
-      @focus="focused = true"
-      @blur="focused = false"
-      @keyup.enter="go(focusIndex)"
-      @keyup.up="onUp"
-      @keyup.down="onDown"
-    />
+  <div :class="searchClasses">
+    <h1 class="search-title">
+      <icon
+        name="chevron-right"
+        class="icon-back"
+        @click.native="$emit('search-toggle', false)"
+      />
+      Search
+    </h1>
+
+    <div class="search-bar">
+      <input
+        id="main-search-field"
+        @input="query = $event.target.value"
+        aria-label="Search"
+        :value="query"
+        autocomplete="off"
+        spellcheck="false"
+        @focus="$emit('focus-toggle', true)"
+        @blur="$emit('focus-toggle', false)"
+        @keyup.enter="go(focusIndex)"
+        @keyup.up="onUp"
+        @keyup.down="onDown"
+        placeholder="Search"
+      />
+      <icon name="search" class="icon-search-field" />
+    </div>
+
+    <div v-if="blankState && isSearchFocused" class="blank-state">
```

Comment:
Should this only display if search is focused? Seems we could display this whether or not the search input is focused.

Reply:
This was a desktop-first approach and has been changed to be hidden with CSS instead when the search isn't focused